src/Testing/MockRequest.php

```diff
@@ -33,7 +32,7 @@ class MockRequest extends \Google\Protobuf\Internal\Message
      * }
      */
     public function __construct($data = NULL) {
-        \GPBMetadata\ApiCore\Testing\Mocks::initOnce();
+        Mocks::initOnce();
```

Comment:
Same here, this is a generated class so it'd be better to just ignore it, as these changes will be overwritten in a future update.